docs/source/operationalguide.rst

```diff
@@ -13,4 +13,5 @@ This section defines the pre-requisites installation and steps for setup of a DL
     operations/setting_dlt
     operations/adding_new_org_corda
     operations/adding_cordapps
-    operations/upgrading_chaincode.md
\ No newline at end of file
+    operations/upgrading_chaincode.md
+    operations/mac_minikube_setup.md
```

Comment:
I did a mistake, this link should be in developerguide.rst and not here. Please add a new section in the developerguide.rst and add the link there.